src/pages/Autolending/LoanTermDropdown.vue

```diff
@@ -4,19 +4,19 @@
 			Loan term
 		</h3>
 		<kv-dropdown-rounded v-model="loanTerm">
-			<option value="default">
+			<option value="70">
```

Comment:
@emuvente 
I set this 'All loans' option to 70 because that's a little over the max value we currently have as a lenderLimit. I can't imagine we'll be doing many loans with a larger lenderLimit than 70. What do you think?